hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/states/NodeStateMap.java

```diff
@@ -445,6 +465,11 @@ private List<DatanodeInfo> filterNodes(Predicate<DatanodeInfo> filter) {
     return result;
   }
 
+  private @Nonnull DatanodeInfo getNodeInfoUnsafe(@Nonnull UUID uuid) throws NodeNotFoundException {
+    checkIfNodeExist(uuid);
+    return nodeMap.get(uuid);
+  }
```

Comment:
I'm not sure it's worth creating this method just to skip lock/unlock where we already have acquired the lock, since it is `Reentrant...`.

But if you prefer to have it, please avoid duplication by calling this one from `getNodeInfo`.